.github/workflows/scripts/prepare-changelog.sh

```diff
@@ -50,17 +50,17 @@ convert_pr_issue_links() {
     local result="$content"
     
     # Find all (#number) patterns
-    local numbers=$(echo "$content" | grep -oE '\(#[0-9]+\)' | grep -oE '[0-9]+' | sort -u)
-    
+    local numbers=$(echo "$content" | grep -oE '\(#[0-9]+((, #[0-9]+)?)*\)' | grep -oE '[0-9]+' | sort -u)
```

Comment:
The change here is to capture optional groups of `, #number` after the first required group. The regex captures as many as it can find, I tested 1 issue up to 4 to ensure it worked as expected.